.github/workflows/release.yml

```diff
@@ -0,0 +1,73 @@
+name: release
+
+on:
+  push:
+    tags:
+      - v*
+
+jobs:
+  release:
+
+    runs-on: ubuntu-latest
+
+    container:
+      image: analysiscenter1/ds-py3:cpu
+
+    steps:
+    - uses: actions/checkout@v1
+
+    - name: Build docs
+      run: |
+        export LC_ALL="C"
+        pip3 install --upgrade sphinx sphinx-rtd-theme
+        git checkout --orphan gh-pages
+        rm -rf *
+        git fetch
+        git checkout origin/master -- batchflow docs
+        cd docs
+        make html
+
+    - name: Commit docs
+      run: |
+        mv docs/_build/html/* .
+        rm -rf docs batchflow
+        touch .nojekyll
+        git config user.email rhudor@gmail.com
+        git config user.name "Roman Kh"
+        git add -A .
+        git commit -am "Build docs"
+
+    - name: Push docs
+      uses: ad-m/github-push-action@master
+      with:
+        github_token: ${{ secrets.GITHUB_TOKEN }}
+        branch: gh-pages
+        force: True
+
+    - name: Clear docs
+      run: rm -rf *
+
+    - uses: actions/checkout@v1
+
+    - name: Generate coverage report
+      run: |
+        pip3 install -U pytest-cov
+        pytest -m "not slow" --cov=./ --cov-report=xml
+
+    - name: Upload coverage to Codecov
+      run: |
+        pip3 install -U codecov
+        codecov -t ${{ secrets.CODECOV_TOKEN }}
+
+    - uses: actions/checkout@v1
+
+    - name: Install dependencies
+      run: pip3 install --upgrade setuptools wheel twine
+
+    - name: Build and publish
+      env:
+        TWINE_USERNAME: ${{ '__token__' }}
+        TWINE_PASSWORD: ${{ secrets.PYPI_TOKEN }}
+      run: |
+        python3 setup.py sdist bdist_wheel
+        twine upload --repository testpypi dist/*
```

Comment:
Not test!

Reply:
This is done on purpose. Since the workflow can only be truly tested, when merged into `master`, I thought it will be a good idea to use [test.pypi.org](test.pypi.org) first.

Reply:
Testing should be done before publishing a release